assignments/prototypes.js

```diff
@@ -14,97 +13,173 @@
-  console.log(swordsman.hp); // 15
-  console.log(mage.name); // Bruce
-  console.log(swordsman.faction); // The Round Table
-  console.log(mage.weapons); // Staff of Shamalama
-  console.log(archer.language); // Elvish
-  console.log(archer.greet()); // Lilith offers a greeting in Elvish.
-  console.log(mage.takeDamage()); // Bruce took damage.
-  console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
-*/
+const swordsman = new Humanoid({
+  createdAt: new Date(),
+  dimensions: {
+    length: 2,
+    width: 2,
+    height: 2,
+  },
+  hp: 15,
+  name: 'Sir Mustachio',
+  faction: 'The Round Table',
+  weapons: [
+    'Giant Sword',
+    'Shield',
+  ],
+  language: 'Common Toungue',
+});
+
+const archer = new Humanoid({
+  createdAt: new Date(),
+  dimensions: {
+    length: 1,
+    width: 2,
+    height: 4,
+  },
+  hp: 10,
+  name: 'Lilith',
+  faction: 'Forest Kingdom',
+  weapons: [
+    'Bow',
+    'Dagger',
+  ],
+  language: 'Elvish',
+});
+
+// console.log(mage.createdAt); // Today's date
+// console.log(archer.dimensions); // { length: 1, width: 2, height: 4 }
+// console.log(swordsman.hp); // 15
+// console.log(mage.name); // Bruce
+// console.log(swordsman.faction); // The Round Table
+// console.log(mage.weapons); // Staff of Shamalama
+// console.log(archer.language); // Elvish
+// console.log(archer.greet()); // Lilith offers a greeting in Elvish.
+// console.log(mage.takeDamage()); // Bruce took damage.
+// console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
+
+// Stretch task: 
+// * Create Villian and Hero classes that inherit from the Humanoid class.  
+// * Give the Hero and Villians different methods that could be used to remove health points from objects which could result in destruction if health gets to 0 or drops below 0;
+// * Create two new objects, one a villian and one a hero and fight it out with methods!
+const Villian = function(obj) {
+  Humanoid.call(this, obj); 
+}
+
+const Hero = function(obj) {
+  Humanoid.call(this, obj);
+}
+
+Villian.prototype = {...Humanoid.prototype}
+ let test = new Villian({
+  name: 'James',
+  hp: 42,
+  createdAt: 'Today',
+  dimensions: 10,
+  faction: 'English',
+  weapons: 'Bloody Knuckles',
+  language: 'Welsh',
+  victim: 'Sirius'
+});
+
+Hero.prototype = {...Humanoid.prototype}
+ let testTwo = new Hero({
+  name: 'Sirius',
+  hp: 21,
+  createdAt: 'Today',
+  dimensions: 10,
+  faction: 'European',
+  weapons: 'none',
+  language: 'Spanish'
+});
 
-  // Stretch task: 
-  // * Create Villian and Hero classes that inherit from the Humanoid class.  
-  // * Give the Hero and Villians different methods that could be used to remove health points from objects which could result in destruction if health gets to 0 or drops below 0;
-  // * Create two new objects, one a villian and one a hero and fight it out with methods!
\ No newline at end of file
+console.log(test.poke(Hero.victim));
```

Comment:
Consider rewriting this console.log like this: ```console.log(test.poke(testTwo.name));```
Alternatively each Hero and Villain prototype could have a method called takeDamage() that would allow them to reduce their HP if some other entity dealt damage to them.